jest.conf/testUtils.js

```diff
@@ -29,3 +22,56 @@ export const testAfterResize = testFunction => {
   };
   animationFrameId = requestAnimationFrame(assertAfterResize);
 };
+
+export async function renderComponent(page, component, props) {
```

Comment:
Just wonder about the organization of these functions, because if we just read `renderComponent` inside a file called `testUtils`, that doesnt give us any clue that we are using it for visual testing. We could create another utils file for visual testing utilities, or we could rename it. I'll leave it up to you.

Reply:
Well, in that case it'd be better to have another file for visual testing utility functions.